docs/editor_manual/editing_existing_pages.md

```diff
@@ -17,6 +17,35 @@ Here is how you can access the edit screen of an existing page:
 -   You can change the title of the page by clicking into the title field.
 -   When you're typing into a field, help text is often displayed on the right-hand side of the screen.
 
+## Managing Revisions
+
+If you need to go back to retrieve a version of the content you need, click the "history" option from the **More** dropdown menu.
+
+Alternatively, you can access the page history from the top right corner of the page you're editing.
+
+![The page dropdown menu showing the history option](../_static/images/Page_history_dropdown.png)
+
+Once you're on the history page, you'll see:
+
+- Actions that have been done on that particular page.
+- The user that carried out that action.
+- The date/time the action occurred.
+
+You can also apply a filter to find specific versions of your content.
+
+![The actions and filters for page history](../_static/images/page_history_filters.gif)
```

Comment:
No need for you to redo this – but I’d prefer we avoid the GIF. They’re very hard for us to keep up to date. I’ll switch over to a PNG when merging.